tests/test_project_structure.py

```diff
@@ -36,6 +36,9 @@
     'tests/providers/apache/pig/operators/test_pig.py',
     'tests/providers/apache/spark/hooks/test_spark_jdbc_script.py',
     'tests/providers/cncf/kubernetes/operators/test_kubernetes_pod.py',
+    'tests/providers/cncf/kubernetes/operators/test_spark_kubernetes_operator.py',
+    'tests/providers/cncf/kubernetes/hooks/test_kubernetes_hook.py',
+    'tests/providers/cncf/kubernetes/sensors/test_spark_kubernetes_sensor.py',
```

Comment:
Naughty Naughty.

Reply:
I know :( unfortunately it still not possible to mock Kubernetes API in python for unit testing but I'll try to write tests to what I can.

Reply:
Why is this not possible? I think that shouldn't be a problem.

Reply:
you are right! I added tests